### PR TITLE
Mirage graphql metadata

### PR DESCRIPTION
**Pull request by mattgraham1995**
## Description
This PR will add the ember-cli-mirage-graphql addon to navi-data and setup some infrastructure for GraphQL mock data. 
This is a first step before we have more info from the web service team on their GraphQL implementation.

## Proposed Changes

- Add `ember-cli-mirage-graphql` addon
- Add schema for mirage graphql endpoint
- Add Mirage graphql endpoint
- Add one example query definition, but this will almost certainly change soon

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jkusa** [comment]
Is this a general `navi-data` thing?

File: packages/data/addon/mirage/routes/graphql.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import graphQLHandler from '../handlers/graphql';
+
+/**
+ * Method to configure metadata endpoints
+ */
+export default function() {
+  /**
+   * /graphql endpoint
+   */
+  this.post('/graphql', graphQLHandler);
```
Comment (kevinhinterlong) [inline packages/data/addon/mirage/routes/graphql.js]: get?

**Comment by mattgraham1995** [comment]
@jkusa Should we make a separate navi-data-graphql package instead?

**Comment by mattgraham1995** [comment]
Moving this to internal repo

**Pull request closed without merging**